File: KataFizzBuzz.SideBySide/FizzBuzz.SideBySide/FizzBuzz_Translate_Tests.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Xunit;

namespace FizzBuzz.SideBySide
{
    public class FizzBuzz_Translate_Tests
    {
        [Fact]
        public void When_Number_Is_Multiples_Of_Three_Then_Returns_Fizz()
        {
            FizzBuzz fizzbuzz = new FizzBuzz();
            string translation = fizzbuzz.Translate(6);
            Assert.Equal("Fizz", translation);
        }

        [Fact]
        public void When_Number_Is_Multiples_Of_Five_Then_Returns_Buzz()
        {
            FizzBuzz fizzbuzz = new FizzBuzz();
            string translation = fizzbuzz.Translate(10);
            Assert.Equal("Buzz", translation);
        }

        [Fact]
        public void When_Number_Is_Multiples_Of_Three_And_Five_Then_Returns_FizzBuzz()
        {
            FizzBuzz fizzbuzz = new FizzBuzz();
            string translation = fizzbuzz.Translate(15);
            Assert.Equal("FizzBuzz", translation);
        }

        [Fact]
        public void When_Number_Is_No_Multiples_Of_Three_Or_Five_Then_Returns_Number()
        {
            FizzBuzz fizzbuzz = new FizzBuzz();
            string translation = fizzbuzz.Translate(7);
            Assert.Equal("7", translation);
        }
    }
}

File: KataFizzBuzz.Java/src/FizzBuzzTranslateTests.java
import static org.junit.Assert.*;
import org.junit.Test;

public class FizzBuzzTranslateTests {
	@Test
	public void multiples_of_three_translates_to_fizz() {
		FizzBuzz fizzbuzz = new FizzBuzz();
		String translation = fizzbuzz.Translate(6);
		assertEquals("fizz", translation);
	}

	@Test
	public void multiples_of_five_translates_to_buzz() {
		FizzBuzz fizzbuzz = new FizzBuzz();
		String translation = fizzbuzz.Translate(10);
		assertEquals("buzz", translation);
	}

	@Test
	public void multiples_of_three_and_five_translates_to_fizzbuzz() {
		FizzBuzz fizzbuzz = new FizzBuzz();
		String translation = fizzbuzz.Translate(15);
		assertEquals("fizzbuzz", translation);
	}
	
	@Test
	public void no_multiples_of_three_or_five_translates_to_number() {
		FizzBuzz fizzbuzz = new FizzBuzz();
		String translation = fizzbuzz.Translate(7);
		assertEquals("7", translation);
	}
}
